sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -287,6 +287,30 @@ case class Cos(child: Expression) extends UnaryMathExpression(math.cos, "COS")
   """)
 case class Cosh(child: Expression) extends UnaryMathExpression(math.cosh, "COSH")
 
+@ExpressionDescription(
+  usage = """
+    _FUNC_(expr) - Returns inverse hyperbolic cosine of `expr`.
+  """,
+  arguments = """
+    Arguments:
+      * expr - hyperbolic angle
+  """,
+  examples = """
+    Examples:
+      > SELECT _FUNC_(1);
+       0.0
+      > SELECT _FUNC_(0);
+       NaN
```

Comment:
This is the difference from PostgreSQL.
```
postgres=# SELECT acosh(0);
ERROR:  input is out of range
```

Reply:
Do you mean we need to check the range of input and raise exception if it is out of range?

Reply:
Nope~ I'm doing the corner case checking to identify the difference.

Reply:
This difference has already be covered in https://issues.apache.org/jira/browse/SPARK-27923?

Reply:
This one returns `NaN` while the scope of SPARK-27923 is `Spark SQL is NULL`.

`NaN` is different from `NULL` semantically.